web/cypress/support/helpers.ts

```diff
@@ -590,12 +487,24 @@ export const checkNewBusinessProfilePage = ({
     onProfilePage.getHomeBased(!homeBasedQuestion).should("not.be.checked");
   }
 
-  onProfilePage
-    .getLegalStructure()
-    .parent()
-    .find("input")
-    .invoke("prop", "value")
-    .should("contain", companyType);
+  if (companyType !== undefined) {
```

Comment:
I see we were using `companyType` instead of `legalStructure` - it seems preferable to use our domain language and not something new. We could refactor this